client/jira.go

```diff
@@ -120,6 +128,38 @@ func CommentIssue(issueId, comment string) (domain.Issue, error) {
 	return issue, nil
 }
 
+func DoTransition(issueId, transitionId string) error {
```

Comment:
Just use `Transition` all across the board drop `Do`

Reply:
done